examples/Test/Results/002_AL_ORU_R01_NBS_Simplified_1_hl7_translation.fhir

```diff
@@ -0,0 +1,3968 @@
+{
```

Comment:
Same for this message as above, the suffix should be 005

Reply:
Fixed the filename now